clients/js/package.json

```diff
@@ -69,7 +69,8 @@
   "peerDependencies": {
     "@google/generative-ai": "^0.1.1",
     "cohere-ai": "^5.0.0 || ^6.0.0 || ^7.0.0",
-    "openai": "^3.0.0 || ^4.0.0"
```

Comment:
Can we keep the openai versions the same?

Reply:
@itaismith Hm, not sure why/how this was changed, i reverted back.